en/docs/guides/identity-federation/google.md

```diff
@@ -138,81 +183,52 @@ Deploy this sample web app on a web container.
+### Sign in with Google 
 
-### Configuring Google One Tap
+To test the sample:
 
-!!! Note "Enabling Google One Tap for tenants"
-     By default, Google One Tap is enabled for all tenants. If you want to restrict this option to selected tenants, add the following configuration to the `deployment.toml`.
+1.  Go to the following URL on your browser: `http://<TOMCAT_HOST>:<TOMCAT_PORT>/pickup-dispatch`.<br/>
     
-     ``` bash
-        [console.ui]
-        google_one_tap_enabled_tenants = [“carbon.super”,”t.com”]
-     ```
+    For example, `http://localhost.com:8080/pickup-dispatch`.
 
-1. Make sure the **Callback url** of Google authenticator has been added as a “Authorized Callback URL” at the relevant Google Client at Google CLI.
+    ![Pickup-dispatch application]({{base_path}}/assets/img/samples/pickup-dispatch-login.png)
 
-    ```
-    https://<IS_HOST>:<IS_PORT>/commonauth
-    ```
+2.  Click **Login**. 
 
-   ![Google One Tap callback]({{base_path}}/assets/img/samples/google-onetap-callback-url.png)
+    You are redirected to the Google login page.
 
-2. Add the login page domain as the **Authorized Javascript Origins** at the relevant Google Client at Google CLI.
+3.  Click **Sign in with Google** to be redirected to the Google sign-in page.
 
-    ```
-    https://<IS_HOST>:<IS_PORT>
-    ```
-   
-    ![Google One Tap origins]({{base_path}}/assets/img/samples/google-onetap-js-origins-url.png)
+    !!! Note
+        If you have **Google One Tap** enabled for your Google IdP, you will be able to skip this step. See the instructions on [signing in with Google One Tap](#sign-in-with-google-one-tap).
+
+    <img src="{{base_path}}/assets/img/samples/sign-in-google.png" alt="Google login page" width="400">
+  
+4. Select your preferred Google account and sign in with your Google credentials.
 
-3. Keep the following configuration empty to enable Google One Tap for all tenants or add the specific tenant name to enable only for that tenant.
+You are redirected to the sample application's home page.
 
-     ``` bash
-        [console.ui]
-        google_one_tap_enabled_tenants = [“carbon.super”,”t.com”]
-     ```
-   
-4. When a tenant is eligible for Google One Tap, **Google One Tap** toggle will be available in Google authenticator settings at Asgardeo developer portal. By enabling this toggle button, users will see a personalized Google sign-in button with Google profile information on the login page.
+### Sign in with Google One Tap
 
-   ![Google One Tap toggle]({{base_path}}/assets/img/samples/google-onetap-toggle.png)
+Google One Tap is a personalized authentication feature provided by Google. When a browser has an already authenticated google session, a personalized sign-in/sign-up button will appear instead of a conventional Google sign-in button. Note that this feature applies to One Tap-supported web applications.
```

Comment:
```suggestion
Google One Tap is a personalized authentication feature provided by Google. When a browser has an already authenticated Google session, a personalized sign-in/sign-up button will appear instead of a conventional Google sign-in button. Note that this feature applies to One Tap-supported web applications.
```